Makefile

```diff
@@ -50,27 +52,18 @@ ssl/rootCA.key:
 # Following targets are called by the `tests` Docker compose service
 enable-vault-path:
 	vault secrets enable -path=approzium -version=1 kv | true
-seed-vault-host:  # call this with "make seed-vault-host ADDR=foo"
+seed-vault-addr:  # call this with "make seed-vault-host ADDR=foo"
 	echo '{"$(TEST_DBUSER)": $(vault_secret)}' | \
 		vault write approzium/$(ADDR) -
-seed-vault-all-hosts:
+seed-vault-all-addrs:
 	for ADDR in $(TEST_DBADDRS); do \
-		make seed-vault-host ADDR=$$ADDR; \
+		make seed-vault-addr ADDR=$$ADDR; \
 	done
 
 run-testsuite: run-gotests run-pg2tests
 
 run-gotests:
 	cd authenticator && CGO_ENABLED=1 go test -v -race -p 1 ./...
 
-run-pythontests: enable-vault-path seed-vault-all-hosts
+run-pythontests: enable-vault-path seed-vault-all-addrs
 	cd sdk/python && poetry run pytest --workers auto
-
-run-pg2tests: enable-vault-path seed-vault-all-hosts
-	for ADDR in $(TEST_DBADDRS); do \
-		make seed-vault-host ADDR=$$ADDR \
-		echo '###### Testing with DBADDR' $$ADDR 'SSL=ON #####'; \
-		PGSSLMODE=require PSYCOPG2_TESTDB_HOST=$$HOST make -C sdk/python/ test; \
-		echo '###### Testing with DBHOST' $$HOST 'SSL=OFF #####'; \
-		PGSSLMODE=disable PSYCOPG2_TESTDB_HOST=$$HOST make -C sdk/python/ test; \
-	done
```

Comment:
if time allows, we can add this back to our test suite, but we currently don't use it, so i think it is better to deprecate it for now